data/test-nursery.py

```diff
@@ -316,3 +319,21 @@
     " If they don't, the SDC analysis, and their outputs, are lost."
 )
 output = acro.finalise("test_results.json")
+
```

Comment:
No change needed here, but this change might have been better in its own commit to make it easier to revert later on

Reply:
Hmm, I developed the relative path test data and the support for it in tandem. In my head, they belong together?